build.gradle

```diff
@@ -12,6 +12,8 @@ repositories {
 dependencies {
     testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.5.0'
     testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.5.0'
+    implementation 'com.google.code.gson:gson:2.8.9'
```

Comment:
Might be misunderstanding, but I'm seeing `gson:2.8.9` here but there is also a `gson-2.10.1.jar` under `src/`